### ISSUE TITLE
Google API changed from v3 to v4

### ISSUE THREAD

**Issue by swarzynski** [brandonrobertz/rescue-me#2]
Dear Brandon,
unfortunately API of the Google Spreadsheets changed and app do not get data anymore.
Can you help?

https://developers.google.com/sheets/api/guides/migration#v3-api_13
https://developers.google.com/gdata/samples/spreadsheet_sample

**Comment by brandonrobertz** [comment]
Hey! You're using this for something? I haven't messed with this in a long time, but I think I could fix this pretty quickly if you are. I wasn't aware of anyone using it.

**Comment by brandonrobertz** [comment]
I fixed the the code. If you use the latest code everything should work. The changes were the following:
- In `js/app.js` you need to change the end of the URL variable from `/pubhtml` to `/pub?output=csv`
- In `js/app.js` change the `Tabletop.init({...})` part to this new block: https://github.com/brandonrobertz/rescue-me/blob/master/js/app.js#L118-L131

LMK if this works for you!

**Comment by swarzynski** [comment]
Hello!

Thank you for such fast fix :)

> Hey! You're using this for something? I haven't messed with this in a long time, but I think I could fix this pretty quickly if you are. 

Yes, I am. Few years ago, for a small group of people in my city, who were helping in cats adoptions, I changed little the code to serve cats profiles: [link](https://domdlamruczka.pl)

> I wasn't aware of anyone using it.

I think it is smart and useful piece of code. It allows to make no-cost (or little cost - own domain) and easy to use, even for non-technical people, site to help adopting pets (I have added links to Google Photoalbums where users can add more photos and videos of the pet). There are many small groups of people who do that and this tool can be helpful.

After few years I am back to this subject and I want to use it for my own cats adoption project.

Nice work - thank you for it! :)
